coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java

```diff
@@ -725,6 +725,15 @@ protected boolean renderEditForPersonnelAttachment(String personId, DevelopmentP
         return true;
     }
 
+    public String getProposalParentStatus(DevelopmentProposal proposal) {
+        if (proposal.isChild()) {
+            ProposalDevelopmentDocument parentDoc = getProposalHierarchyService().getParentDocument(proposal.getProposalDocument());
```

Comment:
This seems a bit wasteful to get the entire Document just to get the status' description.  I wonder what we can do.

I'd rather see:  

Maybe a custom JPQL DAO?